Proposed/asc057.md

```diff
@@ -0,0 +1,26 @@
+#### **Title**
+Revision to `Compound`
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+#### **Motivation**
+Some of the challenges in representing molecular structures in 2D include the lack of standardized representation with enough structural data, and the lack of standardized
+method to interpret one representation to another without altering the data it represents (as perceived by the end user). We initially chose to represent Compound using `standard InChI`.
+However, the consensus from the primary users suggest that SMILES is more commonly used method of representing molecule, and each time SMILES is interpreted in InChI (or vice versa) there is a chance of
+misinterpretation due to the challenges mentioned above.
+
+#### **Proposal**
+To minimize the chance of altering the molecular data or intent contained in the string, we propose to revise Compound class to utilize more widely used representation of molecule in the form of `absolute SMILES`
+which indicates a canonical SMILES with isotopic and chiral specifications. `absolute SMILES` is standardized, human readable, compact, and represents a specific molecular structure.
+
+Example valid values for different compounds:
+
+| Compound      | Example Value
+| -             | -
+| Methane       | "C"
+| Ethanol       | "CCO"
+| Benzene       | "c1ccccc1"
+
+Note there are multiple ways to represent e.g. ethanol in SMILES including "C(O)C", "OCC", "C-C-O" to name a few. However, the canonical,
+unique representation of ethanol in `absolute SMILES` would be "CCO". There is an algorithm to generate a unique SMILES from any one of these generic SMILES.
```

Comment:
` There is an algorithm to generate a unique SMILES from any one of these generic SMILES.` -> We should probably state the name or link to that algorithm rather than saying there is one